web/package.json

```
@@ -9,7 +9,8 @@
     "build:lambda": "yarn run patch-package && yarn netlify-lambda build src/lambda --config ./webpack.functions.js",
     "test:unit": "vue-cli-service test:unit",
     "lint": "vue-cli-service lint",
-    "i18n:report": "vue-cli-service i18n:report --src './src/**/*.?(js|vue)' --locales './src/locales/**/*.json'"
+    "i18n:report": "vue-cli-service i18n:report --src './src/**/*.?(js|vue)' --locales './src/locales/**/*.json'",
+    "e2e": "cypress open --e2e --browser chrome"
```

@ann-kilzer I think there is a conflict between `yarn serve` (webpack) and cypress both trying to use port 8080. 

```suggestion
    "e2e": "cypress open --e2e --browser chrome --port 9090"
```

---

@ann-kilzer or better set the port in `cypress.config.conf`